language_sentiment_text.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "language_sentiment_text")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+# sample-metadata
+#   title: Analyzing Sentiment
+#   description: Analyzing Sentiment in a String
+#   usage: python3 samples/v1/language_sentiment_text.py [--text_content "I am so happy and joyful."]
+
+# [START language_sentiment_text]
+from google.cloud import language_v1
+from google.cloud.language_v1 import enums
+
+
+def sample_analyze_sentiment(text_content):
+    """
+    Analyzing Sentiment in a String
+
+    Args:
+      text_content The text content to analyze
+    """
+
+    client = language_v1.LanguageServiceClient()
+
+    # text_content = 'I am so happy and joyful.'
+
+    # Available types: PLAIN_TEXT, HTML
+    type_ = enums.Document.Type.PLAIN_TEXT
+
+    # Optional. If not specified, the language is automatically detected.
+    # For list of supported languages:
+    # https://cloud.google.com/natural-language/docs/languages
+    language = "en"
+    document = {"content": text_content, "type": type_, "language": language}
+
+    # Available values: NONE, UTF8, UTF16, UTF32
+    encoding_type = enums.EncodingType.UTF8
+
+    response = client.analyze_sentiment(document, encoding_type=encoding_type)
+    # Get overall sentiment of the input document
+    print(u"Document sentiment score: {}".format(response.document_sentiment.score))
+    print(
+        u"Document sentiment magnitude: {}".format(
+            response.document_sentiment.magnitude
+        )
+    )
+    # Get sentiment for all sentences in the document
+    for sentence in response.sentences:
+        print(u"Sentence text: {}".format(sentence.text.content))
+        print(u"Sentence sentiment score: {}".format(sentence.sentiment.score))
+        print(u"Sentence sentiment magnitude: {}".format(sentence.sentiment.magnitude))
+
+    # Get the language of the text, which will be the same as
+    # the language specified in the request or, if not specified,
+    # the automatically-detected language.
+    print(u"Language of the text: {}".format(response.language))
+
+
+# [END language_sentiment_text]
+
+
+def main():
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--text_content", type=str, default="I am so happy and joyful.")
```

Comment:
As per the [style guide](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md):

```
Beyond PEP8, there are several idioms and style nits we prefer.

Use single quotes (') except for docstrings (which use """).
```

This is a hill I will die on, I have become very used to authoring Python strings preferring single-quotes over double-quotes and it is a convention both highlighted in our authoring guide as well as in older versions of the Google Python style guide (although they recently have loosened up this to just be consistent).

Reply:
We use `black` to format the code in all our library repos, which uses double-quotes unless the string contains a double-quote character.

At some point, I think we should start using `black` everywhere because I hate manually trying to format code to pass flake8.